pylint/checkers/base_checker.py

```diff
@@ -109,10 +111,14 @@ def get_full_documentation(self, msgs, options, reports, doc=None, module=None):
         return result
 
     def add_message(
-        self, msgid, line=None, node=None, args=None, confidence=None, col_offset=None
-    ):
-        if not confidence:
-            confidence = UNDEFINED
+        self,
+        msgid: str,
+        line: Optional[int] = None,
+        node: Optional[nodes.NodeNG] = None,
+        args: Optional[Any] = None,
```

Comment:
`Any` also covers `None`

Reply:
Ah, I did this because of this [mypy flag](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-optional). It didn't mention `Any`. My bad!